setup.py

```diff
@@ -30,5 +30,7 @@
     # flake8 compliance configuration
     enable_flake8=True,  # Enable checks
     fail_on_flake=True,  # Fail builds when checks fail
-    install_requires=[],
+    install_requires=[
+        'websockets>=14.1',
```

Comment:
The latest websockets version is 15.0.1. Let's pin to the exact version instead of setting a minimum version.